packages/governance/test/swingsetTests/contractGovernor/test-governor.js

```diff
@@ -132,9 +132,9 @@ const expectedChangeElectorateLog = [
   'vote outcome: {"changes":{"MalleableNumber":"[299792458n]"}}',
   'updated to {"changes":{"MalleableNumber":"[299792458n]"}}',
   'current value of MalleableNumber is 299792458',
-  'Electorate,MalleableNumber changed in a vote.',
-  'Electorate changed in a vote.',
-  'MalleableNumber changed in a vote.',
+  'subscription updated',
+  'subscription updated',
+  'subscription updated',
```

Comment:
There's no longer any evidence in this transcript that the electorate  changed. I think the test has to save the old electorate, and print something saying whether or not it's the same after the vote.  The voter doesn't currently have visibility before the vote (though perhaps they could squirrel it away in `createVoter`, or `castBallotFor` and check in validate).  The other choice is to do a before-and-after in bootstrap.js.

Reply:
Went with before-and-after: dba994f884bf48c4ed5a2f14e5dd6aef96e2da73